test/CMakeLists.txt

```diff
@@ -2158,6 +2158,18 @@ if( iodaconv_bufr_query_ENABLED )
 
 endif()
 
+  ecbuild_add_test( TARGET  test_${PROJECT_NAME}_balloonsonde
+                    TYPE    SCRIPT
+                    ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                    COMMAND bash
+                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                            netcdf
+                            "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/balloon_sonde_json2ioda.py
+                            -i testinput/balloonsonde_sample_20241001T00Z_PT15M.json
+                            -o testrun/obs.20241001T000000Z_PT15M_balloonsonde.nc4
+                            --sort"
+                            obs.20241001T000000Z_PT15M_balloonsonde.nc4 ${IODA_CONV_COMP_TOL})
```

Comment:
@smaticka it was missing this last line.

Because the majority of these ctests are actually calls to `iodaconv_comp.sh` which is a wrapper for `nccmp` so you generate an output.  This last line has both the file in `testoutput` w/o the directory name and then the tolerance value.